! Patch file for PPSSPP's FragmentShaderGeneratorGLES.cpp to view vertex color attribute

--- FragmentShaderGeneratorGLES.cpp	2019-09-24 19:26:57.053657854 +0200
+++ edit.cpp	2019-09-24 19:27:16.743873450 +0200
@@ -815,6 +815,9 @@
 		WRITE(p, "  gl_FragDepth = z;\n");
 	}
 
+	if (texFunc != GE_TEXFUNC_REPLACE || !doTextureAlpha)
+		WRITE(p, "    %s.rgb = v_color0.rgb;\n", fragColor0);
+
 	WRITE(p, "}\n");
 
 	return true;
